docs/topics/loaders.rst

```diff
@@ -174,600 +88,31 @@ is an example::
 
         # ...
 
-As you can see, input processors are declared using the ``_in`` suffix while
-output processors are declared using the ``_out`` suffix. And you can also
-declare a default input/output processors using the
+In this case, field input and output processors are declared as the item's
+field name followed by a ``_in`` or ``_out`` suffix.
+If there's some routine that's generic and shareable across the fields,
+it can be define as a default processor through
 :attr:`ItemLoader.default_input_processor` and
 :attr:`ItemLoader.default_output_processor` attributes.
 
-.. _topics-loaders-processors-declaring:
-
-Declaring Input and Output Processors
-=====================================
-
-As seen in the previous section, input and output processors can be declared in
-the Item Loader definition, and it's very common to declare input processors
-this way. However, there is one more place where you can specify the input and
-output processors to use: in the :ref:`Item Field <topics-items-fields>`
-metadata. Here is an example::
-
-    import scrapy
-    from scrapy.loader.processors import Join, MapCompose, TakeFirst
-    from w3lib.html import remove_tags
-
-    def filter_price(value):
-        if value.isdigit():
-            return value
-
-    class Product(scrapy.Item):
-        name = scrapy.Field(
-            input_processor=MapCompose(remove_tags),
-            output_processor=Join(),
-        )
-        price = scrapy.Field(
-            input_processor=MapCompose(remove_tags, filter_price),
-            output_processor=TakeFirst(),
-        )
-
->>> from scrapy.loader import ItemLoader
->>> il = ItemLoader(item=Product())
->>> il.add_value('name', [u'Welcome to my', u'<strong>website</strong>'])
->>> il.add_value('price', [u'&euro;', u'<span>1000</span>'])
->>> il.load_item()
-{'name': u'Welcome to my website', 'price': u'1000'}
-
-The precedence order, for both input and output processors, is as follows:
-
-1. Item Loader field-specific attributes: ``field_in`` and ``field_out`` (most
-   precedence)
-2. Field metadata (``input_processor`` and ``output_processor`` key)
```

Comment:
I could be wrong, but it seems this information is absent in new docs.

Reply:
It was moved to `itemloaders`, but the `Item` part was missing.